ros2_batch_job/osx_batch/__init__.py

```diff
@@ -38,6 +38,13 @@ def pre(self):
             os.environ['LANG'] = 'en_US.UTF-8'
         if 'ROS_DOMAIN_ID' not in os.environ:
             os.environ['ROS_DOMAIN_ID'] = '111'
+        if 'OPENSSL_ROOT_DIR' not in os.environ:
+            import subprocess
```

Comment:
The import should be at the top of the file. Only modules which can't be loaded globally or are optional should be imported in a local scope.